gammapy/irf/core.py

```diff
@@ -987,16 +987,20 @@ def downsample(self, factor, axis_name=None, weights=None):
             Downsampling factor.
         axis_name : str
             Axis to downsample. By default, spatial axes are downsampled.
+            It is not recommended to use this function on a PSFMap rad axis.
         weights : `~gammapy.maps.Map`, optional
-            Map with weights downsampling. Default is None.
+            Map with weights downsampling. Default is IRFMap exposure map.
 
         Returns
         -------
         map : `IRFMap`
             Downsampled IRF map.
         """
+        if not weights:
+            weights=self.exposure_map
```

Comment:
I think the error with formatting it here. You could implement this and see
```suggestion
            weights = self.exposure_map
```

Reply:
Great, thank you for the suggestions! That could be the issue!